clients/cobol-lsp-vscode-extension/package.json

```diff
@@ -443,6 +443,11 @@
                     ],
                     "uniqueItems": true
                 },
+                "cobol-lsp.cics.translator": {
+                    "type": "boolean",
+                    "default": "true",
+                    "description": "CICS translator"
```

Comment:
@VitGottwald @zeibura @zimlu02  Any suggestion for this?

Reply:
"Allow CICS translator" ?

Reply:
Enable CICS translator. BTW what is a CICS translator?

Reply:
Updated to "Enable CICS translator"

Reply:
![image](https://user-images.githubusercontent.com/69206564/195134004-f1ee82dc-62a6-4ac6-9bb0-18052bd904c9.png)